HCPInterface/preproc/preproc.py

```diff
@@ -57,3 +57,19 @@ def generate_tagmap(fn, tag, out="{}/meta-{}.json".format(os.getcwd(), TIMESTAMP
     md.write(json.dumps(mdict, indent=4))
     md.close()
     log.debug(f'Generated metadata file {out}')
+
```

Comment:
This file in general could do with some more in-line comments

Reply:
Every function has desc. Python code is pretty descriptive. Not sure what is requested